File: Excel/610 Challenge.py
import pandas as pd
from collections import Counter

path = "610 Uncommon Words in Sentences.xlsx"
input_df = pd.read_excel(path, usecols="A:B", nrows=9)
test_df = pd.read_excel(path, usecols="C", nrows=9)

input_df['count'] = input_df.apply(lambda row: sum(1 for count in Counter((row['Sentence1'] + ' ' + row['Sentence2']).split()).values() if count == 1), axis=1)

print(input_df['count'].equals(test_df['Answer Expected'])) # True

File: Excel/600-699/610/610 Challenge.R
library(tidyverse)
library(readxl)

path = "Excel/610 Uncommon Words in Sentences.xlsx"
input = read_excel(path, range = "A1:B9")
test  = read_excel(path, range = "C1:C9")

count_unique <- function(s1, s2) {
  sum(table(unlist(strsplit(c(s1, s2), "\\s+"))) == 1)
}
         
result = input %>%
  mutate(count = map2_dbl(Sentence1, Sentence2, count_unique)) 

all.equal(result$count, test$`Answer Expected`)
#> [1] TRUE